test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -90,12 +90,35 @@ string ProtoConverter::visit(Literal const& _x)
 	}
 }
 
+void ProtoConverter::consolidateVarDeclsInFunctionDef()
+{
+	m_currentFuncVars.clear();
+	auto &scopes = m_funcVars.back();
```

Comment:
Is it possible that `m_funcVars` is empty here?

Reply:
You are right, will fix it